### PR TITLE
[Bug 1589] [UI] Card title is indented

### LINKED ISSUE #1589: [UI] Card title is indented
**Reserved** for [outreachy](https://www.outreachy.org/) candidates. In order for this to be assigned to you, make sure to complete the first step of the application process outlined in the [project description](https://www.outreachy.org/december-2019-to-march-2020-internship-round/communities/mozilla/#build-a-component-that-can-display-a-json-schema).

---

**To Reproduce**
1. Go to https://taskcluster-ui.herokuapp.com/tasks/eWxW0D38QgOv9WZNKgCgAg

**Actual behavior**

The card title is indented.

Screen Shot 2019-10-11 at 4 28 46 PM

**Expected behavior**

The card title is not indented.

Screen Shot 2019-10-11 at 4 27 39 PM

### PR DESCRIPTION
**Pull request by Karskaya**
Left-padding for Task Card and Task Run change from triple to double

Bugzilla Bug: [1589](https://github.com/taskcluster/taskcluster/issues/1589)

Closes #1589.

### CODE REVIEW

**Comment by Karskaya** [comment]
@helfi92 Please review, I changed left padding for both Task Details and Task Run, am I right?

File: ui/src/components/TaskDetailsCard/index.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ import Link from '../../utils/Link';
 
 @withStyles(theme => ({
   headline: {
-    paddingLeft: theme.spacing.triple,
+    paddingLeft: theme.spacing.double,
     paddingRight: theme.spacing.triple,
```
Comment (helfi92) [inline ui/src/components/TaskDetailsCard/index.jsx]: Let's change this to `theme.spacing.double` for consistency.

File: ui/src/components/TaskRunsCard/index.jsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ const DOTS_VARIANT_LIMIT = 5;
 @withStyles(
   theme => ({
     headline: {
-      paddingLeft: theme.spacing.triple,
+      paddingLeft: theme.spacing.double,
       paddingRight: theme.spacing.triple,
```
Comment (helfi92) [inline ui/src/components/TaskRunsCard/index.jsx]: Same thing here.

File: ui/src/components/TaskDetailsCard/index.jsx (hunk 1 above)
Comment (Karskaya) [reply ui/src/components/TaskDetailsCard/index.jsx]: > Let's change this to `theme.spacing.double` for consistency.

Did you mean right-padding? I've committed with changes, please, review one more time.

**Comment by Karskaya** [comment]
> If you rebase your changes onto the latest master, it will fix these error messages.

It's too late, but my result of rebase.
rebase-result

**Pull request merged**